README.md

```diff
@@ -12,6 +12,15 @@ and has borrowed some wisdom from other similar projects e.g.
 
 ![cert-manager high level overview diagram](/docs/high-level-overview.png)
 
+## Quickstart
+
+> Prebuilt images for cert-manager are made available on Dockerhub.
+
+For the common use-case of automatically issuing TLS certificates to
+Ingress resources, aka a [kube-lego](https://github.com/jetstack/kube-lego)
+replacement, see the [cert-manager nginx ingress quick start
+guide](docs/quick-start/nginx-ingress/).
```

Comment:
I'm thinking we should maybe include this as a bullet point along with the other user guides (perhaps at the top of the list?). Just so users will also see we have other user guides that may be useful to read. Having this here might cause users to jump straight to the quick start guide without realising there are some deeper docs available too.